submissions/annStoyano/html-css-popup/index.html

```diff
@@ -0,0 +1,155 @@
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/news.png" alt="news">
+                                    <figcaption class='item-title'>News</figcaption>
+                                </figure>
+                            </a>
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/maps.png" alt="maps">
+                                    <figcaption class='item-title'>Maps</figcaption>
+                                </figure>
+                            </a>
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/files.png" alt="files">
+                                    <figcaption class='item-title'>Files</figcaption>
+                                </figure>
+                            </a>
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/contacts.png" alt="contacts">
+                                    <figcaption class='item-title'>Contacts</figcaption>
+                                </figure>
+                            </a>
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/search.png" alt="search">
+                                    <figcaption class='item-title'>Search</figcaption>
+                                </figure>
+                            </a>
+                            <a href="#" class="popup-item">
+                                <figure class="popup-item-icon">
+                                    <img src="./popup-icons/photos.png" alt="photos">
+                                    <figcaption class='item-title'>Photos</figcaption>
+                                </figure>
+                            </a>
+                            <a class="link kottans" href="https://github.com/kottans">Even more from Kottans</a>
+                        </section>
+                        
+                    </div>
+                </div>
+            </section>
+            <a href="#" class="bell">
+                <figure class="bell-icon">
+                    <img src="popup-icons/bell.png" alt="bell">
+                </figure>
+            </a>
+            <a href="https://github.com/AnnaStoyano" class="user">
+                <figure class="user-photo">
+                    <img src="popup-icons/favicon.png" alt="kottans">
+                </figure>
+            </a>
+        </nav>
+    </header>
+</body>
+
+</html>
```

Comment:
Add an empty new line. 
You should always leave a blank line at the end of the file.